### ISSUE TITLE
BTPaymentButton requires auto layout

### ISSUE THREAD

**Issue by mickeyreiss** [braintree/braintree_ios#65]
There are two known issues with `BTPaymentButton`:
- auto-layout is required
  - A workaround for apps that do not use auto-layout:
```objc
NSDictionary *views = @{ paymentButton: self.paymentButton };
[self addConstraint:[NSLayoutConstraint constraintsWithVisualFormat:@"[paymentButton(==200)]" options:0 metrics:nil views:views]];
[self addConstraint:[NSLayoutConstraint constraintsWithVisualFormat:@"[paymentButton(==200)]" options:0 metrics:nil views:views]];
```
- `*negative or zero item sizes are not supported in the flow layout*` appears in the Xcode console
  - I believe this console line is harmless.
  - It is caused by an invalid state that occurs transiently during initialization, in which the underlying `UICollectionView` has `CGSizeZero` item size.

We intend to fix these bugs in the near future.

**Comment by mickeyreiss** [comment]
> *negative or zero item sizes are not supported in the flow layout* appears in the Xcode console
> I believe this console line is harmless.
> It is caused by an invalid state that occurs transiently during initialization, in which the underlying UICollectionView has CGSizeZero item size.

This issue is being addressed in #131.

**Comment by burnto** [comment]
Fixed in #131
